### DISCUSSION TITLE
Running "pnpm audit" in multirepo

### DISCUSSION THREAD

**Discussion by pitgrap** [pnpm/pnpm#5649] [Q&A]
Hi, 
we have a multirepo with "shared-workspace-lockfile=false", because we need to build and run the packages separately. And the root package.json doesn't include any dependencies.

If I run "pnpm audit" in this workspace I get the following error message:
`ERR_PNPM_AUDIT_NO_LOCKFILE  No pnpm-lock.yaml found: Cannot audit a project without a lockfile`

If I run `pnpm audit -r`, I get:
`ERROR   ERROR  Unknown option: 'recursive'`

How can I audit my multirepo? Do I need to set an "alias" script in every package and run this script from root?

**Comment by lvqq** [comment]
Try this way: `pnpm -r exec pnpm audit`
